numpy/core/numeric.py

```diff
@@ -1175,9 +1175,42 @@ def tensordot(a, b, axes=2):
     (first) axes of `a` (`b`) - the argument `axes` should consist of
     two sequences of the same length, with the first axis to sum over given
     first in both sequences, the second axis second, and so forth.
+    
+    It is worth noting that the axes of result could be reversed sometimes. 
+    And please refer to the example below:
```

Comment:
The axes are not reversed. At best they're not what you expect. Rather that stating "the result may not be what you expect", it would be better to say "the result shape consists of all the non-contracted axes of the first tensor followed by all the non-contracted axes of the second tensor" or similar